tools/api-markdown-documenter/package.json

```diff
@@ -81,7 +81,7 @@
 		"@fluid-tools/markdown-magic": "file:../markdown-magic",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.26.1",
-		"@fluidframework/eslint-config-fluid": "file:../../common/build/eslint-config-fluid",
+		"@fluidframework/eslint-config-fluid": "^5.1.0",
```

Comment:
If you wanted 5.1.0 (which has import/order rule in place), then you need to use ~5.1.0 as ^ allows 5.2.0 to be used and that does not have the order rule (that is currently being restored).
See pnpm-lock.yaml that shows 5.2.0 was the resolution and is currently in use.
cc: @DLehenbauer, @Josmithr